server/statshouse/statshouse-client.cpp

```diff
@@ -1,164 +1,213 @@
+  transport.metric("kphp_instance_cache_elements")
+    .tag(cluster_name)
+    .tag("stored_with_delay")
+    .write_value(unpack(instance_cache_element_stats.elements_stored_with_delay));
+  transport.metric("kphp_instance_cache_elements")
+    .tag(cluster_name)
+    .tag("storing_skipped_due_recent_update")
+    .write_value(unpack(instance_cache_element_stats.elements_storing_skipped_due_recent_update));
+  transport.metric("kphp_instance_cache_elements")
+    .tag(cluster_name)
+    .tag("storing_delayed_due_mutex")
+    .write_value(unpack(instance_cache_element_stats.elements_storing_delayed_due_mutex));
+  transport.metric("kphp_instance_cache_elements").tag(cluster_name).tag("fetched").write_value(unpack(instance_cache_element_stats.elements_fetched));
+  transport.metric("kphp_instance_cache_elements").tag(cluster_name).tag("missed").write_value(unpack(instance_cache_element_stats.elements_missed));
+  transport.metric("kphp_instance_cache_elements")
+    .tag(cluster_name)
+    .tag("missed_earlier")
+    .write_value(unpack(instance_cache_element_stats.elements_missed_earlier));
+  transport.metric("kphp_instance_cache_elements").tag(cluster_name).tag("expired").write_value(unpack(instance_cache_element_stats.elements_expired));
+  transport.metric("kphp_instance_cache_elements").tag(cluster_name).tag("created").write_value(unpack(instance_cache_element_stats.elements_created));
+  transport.metric("kphp_instance_cache_elements").tag(cluster_name).tag("destroyed").write_value(unpack(instance_cache_element_stats.elements_destroyed));
+  transport.metric("kphp_instance_cache_elements").tag(cluster_name).tag("cached").write_value(unpack(instance_cache_element_stats.elements_cached));
+  transport.metric("kphp_instance_cache_elements")
+    .tag(cluster_name)
+    .tag("logically_expired_and_ignored")
+    .write_value(unpack(instance_cache_element_stats.elements_logically_expired_and_ignored));
+  transport.metric("kphp_instance_cache_elements")
+    .tag(cluster_name)
+    .tag("logically_expired_but_fetched")
+    .write_value(unpack(instance_cache_element_stats.elements_logically_expired_but_fetched));
+
+  using namespace job_workers;
+  const JobStats &job_stats = vk::singleton<SharedMemoryManager>::get().get_stats();
+  transport.metric("kphp_workers_jobs_queue_size").tag(cluster_name).write_value(unpack(job_stats.job_queue_size));
+
+  transport.metric("kphp_workers_job_messages").tag(cluster_name).tag("reserved").write_value(job_stats.messages.count);
+  transport.metric("kphp_workers_job_messages").tag(cluster_name).tag("acquire_fails").write_value(unpack(job_stats.messages.acquire_fails));
+  transport.metric("kphp_workers_job_messages").tag(cluster_name).tag("acquire").write_value(unpack(job_stats.messages.acquired));
+  transport.metric("kphp_workers_job_messages").tag(cluster_name).tag("released").write_value(unpack(job_stats.messages.released));
```

Comment:
As far as I can see, now stats about shared messages is duplicated here and below:
`kphp_workers_job_messages` and `kphp_job_workers_shared_messages`